### PR TITLE
ATC-365 Patient mutations added for favorite and general providers

### PR DESCRIPTION
**Pull request by michaelrbooth**

### CODE REVIEW

File: src/graphql/schemas/custom/commonInputs.graphql [hunk 1]
```diff
@@ -0,0 +1,194 @@
+scalar xhtml
+scalar positiveInt
+"""
+Base StructureDefinition for CodeableConcept Type: A concept that may be defined
+by a formal reference to a terminology or ontology or may be provided by text.
+"""
+input CodeableConcept_Input {
```
Comment (imranq2) [inline src/graphql/schemas/custom/commonInputs.graphql:7]: can we consistent on naming?  this one has upper case "I" in "i"

**Comment by imranq2** [comment]
Perhaps we need to document a naming scheme for inputs so they are consistent?

File: src/graphql/schemas/custom/commonInputs.graphql (hunk 1 above)
Comment (michaelrbooth) [reply src/graphql/schemas/custom/commonInputs.graphql:7]: These types were copied from assymetirk's graphql fhir server, they are autogenerated as far as i know in that app, so i was reluctant to make changes there.

Comment (michaelrbooth) [reply src/graphql/schemas/custom/commonInputs.graphql:7]: Found the one inconsistency and fixed it, it must be a flaw in autogeneration but I will keep an eye out for that going forward.

**Comment by tommymorgan** [comment]
@michaelrbooth, @imranq2 gave a LGTM via Slack: https://icanbwell.slack.com/archives/C01AUNX57S4/p1640208119077700.

**Review by ronnie-the-coder — APPROVED** [review]

**Pull request merged**